mysite/blog/tests/test_admin.py

```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
+from django.test.client import Client
+from django.contrib.auth.models import User
+
+
+class PostAdminTest(TestCase):
+    def test_superuser(self):
+
+        password = "123"
+        test_superuser = User.objects.create_superuser(
+            "karthikeyanvenkat", "karthiktest51@gmail.com", password
+        )

```

Comment:
Use named arguments to be more explicit
```suggestion
        test_superuser = User.objects.create_superuser(
            username="karthikeyanvenkat", email="karthiktest51@gmail.com", password="123"
        )

```

Reply:
Updated as the suggestion and Merged the PR